spec/order_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'order'
+require 'menu'
+
+describe Order do
```

Comment:
These tests use a real `Menu` in the `Order` class. How could you isolate these unit tests from the `Menu` class using test doubles?